Documentation/installation/README.md

```diff
@@ -1,20 +1,25 @@
 # Installation
 
-Clone the git repository and build:
+## Installing the latest stable version
+
+The latest stable version requires go 1.12 or a later version.
```

Comment:
The documentation intentionally omits to specify the minimum version of go because historically we forget to update it when it changes.